test/admin/values.test.js

```diff
@@ -0,0 +1,59 @@
+const StitchMongoFixture = require('../fixtures/stitch_mongo_fixture');
+
+import {getAuthenticatedClient} from './testutil';
+
+describe('Values V2', ()=>{
+  let test = new StitchMongoFixture();
+  let apps;
+  let app;
+  beforeAll(() => test.setup({createApp: false}));
+  afterAll(() => test.teardown());
+  beforeEach(async () =>{
+    let adminClient = await getAuthenticatedClient(test.userData.apiKey.key);
+    test.groupId = test.userData.group.groupId;
+    apps = await adminClient.v2().apps(test.groupId);
+    app = await apps.create({name: 'testname'});
+    appValues = adminClient.v2().apps(test.groupId).app(app._id).values();
+  });
+  afterEach(async () => {
+    await apps.app(app._id).remove();
+  });
+
+  let appValues;
+  const testValueName = 'testvaluename';
+  it('listing values should return empty list', async () => {
+    let values = await appValues.list();
+    expect(values).toEqual([]);
+  });
+  it('creating value should make it appear in list', async () => {
+    let value = await appValues.create({name: testValueName});
+    expect(value.name).toEqual(testValueName);
+    let values = await appValues.list();
+    expect(values).toHaveLength(1);
+    expect(values[0]).toEqual(value);
+  });
+  it('fetching a value returns deep value data', async () => {
+    let value = await appValues.create({name: testValueName, value: '"foo"'});
```

Comment:
[q] Should working with values in the SDK be slightly less raw such that you can pass it any value that we will then extend the JSON on? For example, this test would insert `"foo"`

Reply:
So, this test is a little misleading, because it should just be `'foo'` instead of `'"foo"'` (updated), so it actually does just that - in other words, the body of the request is the result of `JSON.stringify(args)`

Reply:
oh. got it!